serde/src/test/org/apache/hadoop/hive/serde2/typeinfo/TestTypeInfoUtils.java

```diff
@@ -47,7 +47,11 @@ public void testTypeInfoParser() {
         "string",
         "varchar(10)",
         "char(15)",
-        "array<int>"
+        "array<int>",
+        "decimal(10,2)",
+        "decimal(10, 2)",
+        "decimal(10, 2 )",
+        "decimal( 10, 2 )"
```

Comment:
HIVE-11476 added a special case that allowing spaces in some types. e.g `struct<user id:int,user group: int>` but it not added the test case. Could you please add this test case here to make sure we won't get a regression?

BTW, i have tested the case `struct<user id:int,user group: int>` with this change locally and it worked well. But i think it is necessary to add it explicitly.  cc @ayushtkn

Reply:
thanks, added the case.